pkg/storaged/utils.js

```diff
@@ -438,7 +474,10 @@ export function get_available_spaces(client) {
                     block,
                     start: p.start,
                     size: p.size,
-                    desc: cockpit.format(_("unpartitioned space on $0"), text)
+                    desc: cockpit.format(_("unpartitioned space on $0"), text),
+                    type_label: _("unpartitioned space"),
+                    name: link_parts.name,
+                    subname: link_parts.subname
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18967-20230629-132526-4d4c19dd-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)